test/sync/src/androidTest/kotlin/io/realm/test/mongodb/shared/SyncConfigTests.kt

```diff
@@ -109,6 +111,35 @@ class SyncConfigTests {
         assertNotNull(config.errorHandler)
     }
 
+    @Test
```

Comment:
Why are these tests in the `sync` section. Maybe we should actuall test the common parts on both `base` and `sync`? 🤔

Reply:
This is just a smoketest...most tests are in `CompactOnLaunchTests`, but it isn't possible to test SyncConfiguation there.

Right now we lack a good pattern for testing common functionality across `RealmConfiguration` and `SyncConfiguration`...we should probably figure something out, but not sure this PR is the place for it.